angular/src/app/login-page/login-page.component.ts

```diff
@@ -38,11 +37,25 @@ export class LoginPageComponent implements OnInit {
     });
   }
 
-  ngOnInit() {}
+  initGoogle() {
+    window["google"].accounts.id.initialize({
```

Comment:
is there an easy way to add hosted_domain ? (it was not possible with the last lib)

Reply:
according to the documention no